source/app/powertabeditor.cpp

```diff
@@ -1366,6 +1367,28 @@ void PowerTabEditor::editRepeatEnding()
     }
 }
 
+void PowerTabEditor::updateDynamic(Dynamic::VolumeLevel volume)
+{
+    ScoreLocation &location = getLocation();
+    const Dynamic *dynamic = ScoreUtils::findByPosition(
+                location.getStaff().getDynamics(), location.getPositionIndex());
+
+    if (dynamic)
+    {
+        myUndoManager->push(new EditDynamic(location, volume, dynamic),
```

Comment:
For this compiler error I think you just need to replace `dynamic` with `*dynamic`.

Reply:
It looks like that did the trick, but why does dynamic have to be dereferenced there but not in PowerTabEditor::editDynamic when a new AddDynamic is instantiated?

Reply:
oh, in that case there is another variable with the name `dynamic` declared in the inner scope (`Dynamic dynamic(location.getPositionIndex(), ...`), which is the new dynamic that will be inserted. It has type `Dynamic` instead of being a pointer (`Dynamic *`) so it doesn't need to be dereferenced. It would have been more clear if the code named it `new_dynamic` or something like that.

Reply:
I knew it was something simple. I've made some of the variable names more explicit in `updateDynamic` and the `editDynamic` constructor. Compiles now.